docs/how-to-guides/block-tutorial/block-supports-in-dynamic-blocks.md

```diff
@@ -19,6 +19,8 @@ import {
 	InspectorControls,
 } from '@wordpress/block-editor';
 
+import { __ } from '@wordpress/i18n';
```

Comment:
```suggestion
import { __ } from '@wordpress/i18n';
```

To match the rest of the imports, I would remove the empty like above the import. Besides that nitpick this is a good catch  👍 Thanks for the contribution :)